.pre-commit-config.yaml

```diff
@@ -4,12 +4,15 @@ repos:
     hooks:
       - id: black
         stages: [commit]
+        files: ^(backend|tests|scripts)/
   - repo: https://github.com/pycqa/flake8
     rev: 4.0.1 # Must be kept in sync with flake8 version in requirements.txt
     hooks:
       - id: flake8
         stages: [commit]
         files: ^(backend|tests|scripts)/
+        additional_dependencies:
+          - flake8-black==0.3.2
```

Comment:
is this matched in the GHA?

Reply:
Add comment about keeping versions in sync [here](https://github.com/chanzuckerberg/single-cell-data-portal/blob/2a551041c1282c4bb246ad4e328487f3ab60f254/requirements.txt#L8), in the same manner as the neighboring lines.